addons/web/static/src/webclient/actions/client_actions.js

```diff
@@ -41,3 +43,55 @@ class InvalidAction extends Component {
 InvalidAction.template = xml`<div class="o_invalid_action"></div>`;
 
 registry.category("actions").add("invalid_action", InvalidAction);
+
+/**
+ * Client action to reload the whole interface.
+ * If action.params.menu_id, it opens the given menu entry.
+ * If action.params.action_id, it opens the given action.
+ * If action.params.wait, reload will wait the server to be reachable before reloading
+ */
+function reload(env, action) {
+    const { menu_id, action_id, wait } = action.params || {};
+    const { router } = env.services;
+    const route = { ...router.current };
+
+    route.hash = {};
+    if (menu_id) {
+        route.hash.menu_id = menu_id;
+    }
+    if (action_id) {
+        route.hash.action = action_id;
+    }
+
+    const url = browser.location.origin + routeToUrl(route);
+
+    env.bus.trigger("CLEAR-CACHES");
+    router.redirect(url, wait);
+}
+
+registry.category("actions").add("reload", reload);
+
+/**
+ * Client action to go back home.
+ * If action.params.wait, reload will wait the server to be reachable before reloading
+ */
+function home(env, action) {
+    const { wait } = action.params || {};
+    const url = "/" + (browser.location.search || "");
+    env.services.router.redirect(url, wait);
+}
+
+registry.category("actions").add("home", home);
```

Comment:
couldn't find any code calling this, is it actually used?

Reply:
grepped the following:
doAction.*home  (case sensitive)
action.*home (case sensitive)
home (case sensitive, whole word) => found ~70 results in .js files, none of which seemed to call this

Reply:
simply grep `'home'`

Reply:
the first match is (probably) the only usecase

Reply:
ha, yes - forgot to grep on the python side as well. Never mind then :)